riscemu/types/immediate.py

```diff
@@ -0,0 +1,17 @@
+from abc import ABC
+from enum import Enum
+
+class ImmediateKind(Enum):
+    integer = 0
+    label = 1
+
+class Immediate(ABC):
+    value: int
+    type: ImmediateKind
+
+    def __init__(self, value, type):
+        self.value = value
+        self.type = type
+
+    
+    
```

Comment:
I wonder why github shows a (-) here. Are you using a naughty editor that inserts extra characters around `\n`s?